ak_vendor/report/report.py

```diff
@@ -937,6 +953,37 @@ def _count_severity(self, label: str) -> int:
                 count += 1
         return count
 
+    def _count_severity_overridden(self, label: str) -> str:
```

Comment:
A small suggestion. Would not it be better to pass the RiskEnum value to compare against overridden risk

Reply:
We can do that, i only followed the existing approach used. Will push with that change